### DISCUSSION TITLE
Eliminar el uppercase en pretitles y title 1

### DISCUSSION THREAD

**Discussion by aweell** [Telefonica/mistica-design#958] [Proposals]
Se propone dejar de utilizar el uppercase en los siguiente casos:

* Title 1
* Pretitles de cards y otros componentes en los que se esté usando actualmente

**Comment by AnaMontes11** [comment]
Comentamos que el Title 1 en uppercase estaría bien mantenerlo dado que su uso es muy útil para crear jerarquías en el contenido y separar a nivel visual.

**Comment by aweell** [comment]
[Update 24/01/23] Realizar pruebas del pretitle en sentence case + posible subida de preset + posible subida de peso
